dev-python/pyilmbase/pyilmbase-2.2.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1 cmake-utils
+
+DESCRIPTION="IlmBase Python bindings"
+HOMEPAGE="http://www.openexr.com"
+SRC_URI="http://download.savannah.gnu.org/releases/openexr/${P}.tar.gz"
+LICENSE="BSD"
+
+SLOT="0/23" # based on SONAME
+KEYWORDS="~x86 ~amd64"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	>=dev-libs/boost-1.62.0-r1:=[python(+),${PYTHON_USEDEP}]
+	>=dev-python/numpy-1.10.4
+	~media-libs/ilmbase-${PV}:="
+
+DEPEND="${RDEPEND}
+	>=virtual/pkgconfig-0-r1"
+
+RESTRICT="test"
```

Comment:
Please add an explicit explanation why tests are disabled (as a comment).

Reply:
This was based on the original ebuild, so I don't know the answer to this.

Reply:
Could you try running them then?

Reply:
I can try. I know for a fact that IlmBase and OpenEXR tests are seriously broken with bug reports both upstream and in b.g.o. I need to double check I put those links in the ebuilds.